File: app/src/main/java/com/example/colorpicker/ColorPickerViewModel.java
package com.example.colorpicker;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ColorPickerViewModel extends ViewModel {
    MutableLiveData<String> selectedColor = new MutableLiveData<>();

    public LiveData<String> getSelectedColor() {
        return selectedColor;
    }

    public void setSelectedColor(String color) {
        selectedColor.setValue(color);
    }

}

File: app/src/main/java/com/example/colorpicker/ColorPickerViewModel.kt
package com.example.colorpicker

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ColorPickerViewModel : ViewModel() {

    private val _selectedColor = MutableLiveData("#FF000000")
    val selectedColor: LiveData<String> get() = _selectedColor

    fun setSelectedColor(color: String) {
        _selectedColor.value = color
    }
}
